core-httpclient-impl/src/main/java/com/optimizely/ab/config/HttpProjectConfigManager.java

```diff
@@ -125,14 +130,30 @@ protected ProjectConfig poll() {
         return null;
     }
 
+    HttpGet createHttpRequest() {
```

Comment:
Should `createHttpRequest` be `private`?

Reply:
Private must be right one. This default access will be useful for header testing.

Reply:
Then let's use the [VisibleForTesting annotation](https://github.com/optimizely/java-sdk/blob/89d18ac95bf71f0942d89e6363785d3ea5f79b7b/core-api/src/main/java/com/optimizely/ab/annotations/VisibleForTesting.java#L22).